tests/endpoints/events/biosero/test_source_all_negatives.py

```diff
@@ -0,0 +1,203 @@
+from http import HTTPStatus
```

Comment:
this file can go too, event source_no_pickable_samples is the one we will use

Reply:
ok